browser-checklist.md

```diff
@@ -0,0 +1,26 @@
+## Browsers
+
+- [ ] Chrome
+- [ ] Safari
+- [ ] FF
+- [ ] IE10
+- [ ] IE9
+- [ ] IE8
+- [ ] IE7
+- [ ] Opera
+- [ ] iOS8
+- [ ] Android 5
+- [ ] Blackberry Bold
+
+## Accessibility
+
+- [ ] Keyboard friendly
+- [ ] Screen reader friendly
+
+## Other
+
+- [ ] Is useable without CSS
+- [ ] Is useable without JS
+- [ ] Flexible from small to large screens
+- [ ] No linting errors or warnings
```

Comment:
Warning are okay I think, that's the point of having errors _and_ warning level linting, anything that absolutely should not pass should be at an error level.